components/slider.json

```diff
@@ -0,0 +1,44 @@
+{
+  "slider": {
+    "comment": "applies to sliders",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=46518%3A104257",
+    "track-empty": {
+      "#normal": {
+        "background": "@theme-control-inactive-normal"
+      },
+      "#disabled": {
+        "background": "@theme-control-inactive-disabled"
+      }
+    },
+    "track-full": {
+      "#normal": {
+        "background": "@theme-control-active-normal"
+      },
+      "#disabled": {
+        "background": "@theme-control-active-disabled"
+      }
+    },
+    "handle": {
+      "#normal": {
+        "background": "@theme-background-solid-primary-normal",
+        "border": "@theme-outline-primary-normal"
+      },
+      "#hovered": {
+        "background": "@theme-background-solid-secondary-normal",
+        "border": "@theme-outline-primary-normal"
+      },
+      "#pressed": {
+        "background": "@theme-background-solid-secondary-normal",
+        "border": "@theme-outline-primary-normal"
+      },
+      "#focused": {
+        "background": "@theme-background-solid-primary-normal",
+        "border": "@theme-control-active-normal"
+      },
+      "#disabled": {
+        "background": "@theme-background-solid-tertiary-normal",
+        "border": "@theme-background-solid-tertiary-normal"
```

Comment:
you can put "border": "none". "none" is a keyword that makes the "background" colour of that object be assigned to its "border". The output is the same as you have here, but it is just more readable that this handle has no border on disabled.